public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js

```diff
@@ -248,8 +248,17 @@ function Onboarding(svl, audioEffect, compass, form, handAnimation, mapService,
     function drawDoubleClickIcon(x, y) {
         // https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/drawImage
         var image = document.getElementById("double-click-icon");
+        var line1 = document.getElementById("double-click-overlay-text-1").innerHTML;
+        var line2 = document.getElementById("double-click-overlay-text-2").innerHTML;
+
         ctx.save();
         ctx.drawImage(image, x - 50, y - 50, 100, 100);
+        
+        ctx.font = 'bold 14px sans-serif';
```

Comment:
Is it possible to put the font and text align attributes in the CSS file? Or does that not work and you need it in the JavaScript?

Reply:
I have always seen it done this way. From my understanding, JS canvases have their own slightly different styling from CSS. There is technically a way to move this styling to the .css file but it is convoluted and requires some odd indirection moving info from CSS->JavaScript->Canvas. I think this is the best way, as I just read that the process of doing that can lead to browser incompatibility too.